src/app/api/comment/[id]/route.ts

```diff
@@ -0,0 +1,5 @@
+import { withErrorHandler } from '../../core/utils/withErrorHandler'
```

Comment:
absolute import

Reply:
Updated in 8315c817d2e398c277c36edbadab25a14b0e42fd commit